### DISCUSSION TITLE
Adding author image

### DISCUSSION THREAD

**Discussion by Piotr-GitHub** [advplyr/audiobookshelf#481] [Q&A] [answered]
Hi,

I've updated today the server (docker) with new build. I noticed that there is authors section. It pulls nicely data for English authors but if I have a person of different nationality that cannot be updated automatically, how do I add image manually? I see only delete button.

Thanks in advance

**Comment by advplyr** [comment, marked as answer]
Manually adding author images is still in progress

**Comment by troycarpenter** [comment]
Looking forward to this.  I also use the docker image, but it only matched a handful of my authors, English or not.
